File: library/src/main/java/com/ethanhua/skeleton/Skeleton.java
package com.ethanhua.skeleton;

import android.view.View;

import androidx.recyclerview.widget.RecyclerView;

/**
 * Created by ethanhua on 2017/7/29.
 */

public class Skeleton {

    public static RecyclerViewSkeletonScreen.Builder bind(RecyclerView recyclerView) {
        return new RecyclerViewSkeletonScreen.Builder(recyclerView);
    }

    public static ViewSkeletonScreen.Builder bind(View view) {
        return new ViewSkeletonScreen.Builder(view);
    }

}

File: library/src/main/java/com/ethanhua/skeleton/Skeleton.kt
package com.ethanhua.skeleton

import android.support.v7.widget.RecyclerView
import android.view.View

/**
 * Created by ethanhua on 2017/7/29.
 */
object Skeleton {
    @JvmStatic
    fun bind(recyclerView: RecyclerView): RecyclerViewSkeletonScreen.Builder {
        return RecyclerViewSkeletonScreen.Builder(recyclerView)
    }
    @JvmStatic
    fun bind(view: View): ViewSkeletonScreen.Builder {
        return ViewSkeletonScreen.Builder(view)
    }
}